mvt/feature.go

```diff
@@ -4,9 +4,8 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/go-spatial/geom"
 	"github.com/go-spatial/geom/encoding/wkt"
-	"github.com/go-spatial/tegola"
-	"github.com/go-spatial/tegola/internal/convert"
 	"github.com/go-spatial/tegola/internal/log"
```

Comment:
let's remove all instances of the tegola internal logger so we can position this for moving to the geom package.